tfjs-backend-webgpu/src/webgpu_util.ts

```diff
@@ -58,6 +58,19 @@ export function computeDispatch(
   ];
 }
 
+export function computeWorkPerThreadForMatMul(
+    layout: {x: number[], y?: number[], z?: number[]},
+    outputShape: number[],
+    workGroupSize: [number, number, number],
+    workPerThread: number): number {
+  const dispatch = computeDispatch(layout, outputShape, workGroupSize,
+      [workPerThread, workPerThread, 1]);
+  if (arrayProduct(dispatch) === 1) {
```

Comment:
Can you add a comment to explain why the WPT should be 1 when only dispatching one work group? And add a test case to show the perf gain?

Reply:
Done. PTAL

Reply:
Hi  Yunfei, your comments confused me. Maybe your want to express that 'If dispaching number is one, it means only one work group is running. For modern GPUs, it supports multiple work groups running in parallel. So there may be some idle hardware threads. In this case, we prefer to reduce the work per thread and improve the thread utilization'. If you think it's better, feel free to use it or make some changes based on it.

Reply:
+1 to @qjia7's comment!

Reply:
Done. PTAL